src/theory/arith/theory_arith_private.cpp

```diff
@@ -3937,15 +3937,33 @@ Node TheoryArithPrivate::branchIntegerVariable(ArithVar x) const {
   TNode var = d_partialModel.asNode(x);
   Integer floor_d = d.floor();
 
-  //Node eq = Rewriter::rewrite(NodeManager::currentNM()->mkNode(kind::EQUAL, var, mkRationalNode(floor_d+1)));
-  //Node diseq = eq.notNode();
-
-  Node ub = Rewriter::rewrite(NodeManager::currentNM()->mkNode(kind::LEQ, var, mkRationalNode(floor_d)));
-  Node lb = ub.notNode();
-
+  Node lem;
```

Comment:
Add `NodeManger *nm = NodeManager::currentNM();` here and use `nm` instead of calling `NodeManager::currentNM()` each time.